### ISSUE TITLE
Wrong mouse cursor on the note pane scrollbar

### ISSUE THREAD

**Issue by virtuallyvlad** [standardnotes/desktop#719]
On both the Windows desktop app and the web app, the mouse cursor when using the text scrollbar is wrong: it has the "text select" style cursor instead of the default/pointer cursor.

Meanwhile, the cursors on the other scrollbars, the left/tag pane and the middle/list pane, correctly use the default/pointer cursor.

**Comment by JaspalSuri** [comment]
Hi @virtuallyvlad, thank you for bringing this issue to our attention. I can reproduce it in the web app but not in the desktop app on Windows 10 in the latest beta (v3.9.0b1) can you export a set of backups of your notes via the Account menu, [install the beta,](https://github.com/standardnotes/desktop/releases/tag/v3.9.0-beta01) and verify whether the issue persists?

**Comment by virtuallyvlad** [comment]
Yup, it still does it in v3.9.0b1.  Tested that beta on win7 and win10; both tests had the wrong cursor over that scrollbar.

**Comment by JaspalSuri** [comment]
>FYI this is with the Plain Editor.

Thank you for sharing that! I was able to reproduce the issue. 

>Yup, it still does it in v3.9.0b1. Tested that beta on win7 and win10; both tests had the wrong cursor over that scrollbar.

Thank you for testing the beta app as well. Please update to the latest stable (in order to receive future updates automatically) from [here](https://standardnotes.org/download). 

>Also, perhaps this should be moved to [the forum](https://github.com/standardnotes/forum/issues) as it applies to both desktop & web.

In this case, if it affects both apps, then it belongs in the web app's repo as the web app precedes the desktop app in development. 🙂
